### ISSUE TITLE
Move Client to Qt.6.4

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#4986]
We should consider moving the client to Qt 6.4 as thats out now

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-3321)
┆Reporter: Basti

**Comment by data-sync-user** [comment]
➤ Matt Cleinman commented:

Unblocking this by removing https://mozilla-hub.atlassian.net/browse/VPN-3385 ( https://mozilla-hub.atlassian.net/browse/VPN-3385|smart-link ) as a blocker, as nobody seems to know why it’s listed as such.

**Comment by data-sync-user** [comment]
➤ Beatriz Rizental Machado commented:

Putting this on hold. It is a rabbit hole and in order to attempt to unblock everything that is blocked by a Qt update wea re going back to https://mozilla-hub.atlassian.net/browse/VPN-2458 ( https://mozilla-hub.atlassian.net/browse/VPN-2458|smart-link )

**Comment by data-sync-user** [comment]
➤ Beatriz Rizental Machado commented:

We are moving directly to 6.5.
